### DISCUSSION TITLE
Apollo Server 3

### DISCUSSION THREAD

**Discussion by flashtheman** [accounts-js/accounts#1176] [Q&A]
Hi!
I just wanted to ask if there are plans to upgrade the backend from Apollo 2 to Apollo 3?
Thanks

**Comment by flashtheman** [comment]
FYI: I was asking because Node 14 doesn't work in combination with Apollo Server 2 in regards to uploading files via graphql.

**Reply by pradel** [reply]
yes @darkbasic started working on a pr that will add support for Apollo v3 https://github.com/accounts-js/accounts/pull/1189

**Reply by darkbasic** [reply]
Tomorrow we will release the first release candidate of version 1.0 that will support apollo v3: https://github.com/accounts-js/accounts/pull/1258
